owais635/planner_flutter
# Added Email Verification Functionality (#12)

Issue Reference: #12 

### Details
- Added email verification feature. Every new user who signs up via email and password (not via Google Sign-in), must verify their email before being allowed to log in.

### Manual Testing
- For testing purposes, a temporary email is recommended, however not required.
1) Click on the sign-in via email button.
2) Enter the required information in the sign-in form to create a new account
3) At this point once the account is created, you should be redirected to the home screen and a snackBar should instruct you to verify your email.
     3b) Attempt to sign in without verifying your email. A snackBar should remind you to verify your email before proceeding to log in
4) Verify your email by going to your email and clicking on the link, your email should now be verified
5) Attempt to sign in, you should be able to successfully sign in now

https://user-images.githubusercontent.com/44068314/125151538-e6a37680-e114-11eb-909b-10be29cce01d.mov
